I0402 10:38:12.717686       1 version.go:38] Welcome to TiDB Operator.
I0402 10:38:12.717704       1 version.go:39] TiDB Operator Version: version.Info{GitVersion:"v1.3.2", GitCommit:"019c90c96c748ed6e5bdee2c3b7594a6c211f32a", GitTreeState:"clean", BuildDate:"2022-03-17T16:23:53Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
I0402 10:38:12.717732       1 main.go:75] FLAG: --V="false"
I0402 10:38:12.717738       1 main.go:75] FLAG: --add_dir_header="false"
I0402 10:38:12.717742       1 main.go:75] FLAG: --alsologtostderr="false"
I0402 10:38:12.717758       1 main.go:75] FLAG: --auto-failover="true"
I0402 10:38:12.717761       1 main.go:75] FLAG: --cluster-permission-node="true"
I0402 10:38:12.717764       1 main.go:75] FLAG: --cluster-permission-pv="true"
I0402 10:38:12.717766       1 main.go:75] FLAG: --cluster-permission-sc="true"
I0402 10:38:12.717769       1 main.go:75] FLAG: --cluster-scoped="true"
I0402 10:38:12.717772       1 main.go:75] FLAG: --dm-master-failover-period="5m0s"
I0402 10:38:12.717776       1 main.go:75] FLAG: --dm-worker-failover-period="5m0s"
I0402 10:38:12.717778       1 main.go:75] FLAG: --features="AdvancedStatefulSet=false,AutoScaling=false,StableScheduling=true"
I0402 10:38:12.717787       1 main.go:75] FLAG: --leader-lease-duration="15s"
I0402 10:38:12.717790       1 main.go:75] FLAG: --leader-renew-deadline="10s"
I0402 10:38:12.717793       1 main.go:75] FLAG: --leader-retry-period="2s"
I0402 10:38:12.717795       1 main.go:75] FLAG: --log_backtrace_at=":0"
I0402 10:38:12.717800       1 main.go:75] FLAG: --log_dir=""
I0402 10:38:12.717803       1 main.go:75] FLAG: --log_file=""
I0402 10:38:12.717806       1 main.go:75] FLAG: --log_file_max_size="1800"
I0402 10:38:12.717815       1 main.go:75] FLAG: --logtostderr="true"
I0402 10:38:12.717818       1 main.go:75] FLAG: --pd-failover-period="5m0s"
I0402 10:38:12.717821       1 main.go:75] FLAG: --resync-duration="30s"
I0402 10:38:12.717824       1 main.go:75] FLAG: --selector=""
I0402 10:38:12.717826       1 main.go:75] FLAG: --skip_headers="false"
I0402 10:38:12.717829       1 main.go:75] FLAG: --skip_log_headers="false"
I0402 10:38:12.717832       1 main.go:75] FLAG: --stderrthreshold="2"
I0402 10:38:12.717835       1 main.go:75] FLAG: --test-mode="false"
I0402 10:38:12.717838       1 main.go:75] FLAG: --tidb-backup-manager-image="pingcap/tidb-backup-manager:v1.3.2"
I0402 10:38:12.717842       1 main.go:75] FLAG: --tidb-discovery-image="pingcap/tidb-operator:v1.3.2"
I0402 10:38:12.717845       1 main.go:75] FLAG: --tidb-failover-period="5m0s"
I0402 10:38:12.717848       1 main.go:75] FLAG: --tiflash-failover-period="5m0s"
I0402 10:38:12.717851       1 main.go:75] FLAG: --tikv-failover-period="5m0s"
I0402 10:38:12.717854       1 main.go:75] FLAG: --v="2"
I0402 10:38:12.717857       1 main.go:75] FLAG: --version="false"
I0402 10:38:12.717860       1 main.go:75] FLAG: --vmodule=""
I0402 10:38:12.717864       1 main.go:75] FLAG: --workers="5"
I0402 10:38:12.717888       1 main.go:92] HELM_RELEASE environment variable not set
I0402 10:38:12.746119       1 leaderelection.go:243] attempting to acquire leader lease  acto-namespace/tidb-controller-manager...
I0402 10:38:30.176729       1 leaderelection.go:253] successfully acquired lease acto-namespace/tidb-controller-manager
I0402 10:38:30.180771       1 upgrader.go:109] Upgrader: APIGroup apps.pingcap.com is not registered, skip checking Advanced Statfulset
I0402 10:38:30.181377       1 reflector.go:207] Starting reflector *v1alpha1.BackupSchedule (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181408       1 reflector.go:207] Starting reflector *v1alpha1.TidbClusterAutoScaler (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181414       1 reflector.go:207] Starting reflector *v1alpha1.TidbNGMonitoring (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181459       1 reflector.go:207] Starting reflector *v1alpha1.Backup (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181481       1 reflector.go:207] Starting reflector *v1alpha1.TidbCluster (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181387       1 reflector.go:207] Starting reflector *v1alpha1.TidbMonitor (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181505       1 reflector.go:207] Starting reflector *v1alpha1.TidbInitializer (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181578       1 reflector.go:207] Starting reflector *v1alpha1.Restore (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.181663       1 reflector.go:207] Starting reflector *v1alpha1.DMCluster (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281607       1 reflector.go:207] Starting reflector *v1.Deployment (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281671       1 reflector.go:207] Starting reflector *v1.Node (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281717       1 reflector.go:207] Starting reflector *v1.StatefulSet (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281728       1 reflector.go:207] Starting reflector *v1.Ingress (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281727       1 reflector.go:207] Starting reflector *v1.Pod (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281673       1 reflector.go:207] Starting reflector *v1.Endpoints (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281787       1 reflector.go:207] Starting reflector *v1.Service (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281671       1 reflector.go:207] Starting reflector *v1.PersistentVolumeClaim (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281890       1 reflector.go:207] Starting reflector *v1.Job (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281723       1 reflector.go:207] Starting reflector *v1.StorageClass (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281748       1 reflector.go:207] Starting reflector *v1.PersistentVolume (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.281608       1 reflector.go:207] Starting reflector *v1.Secret (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.381767       1 reflector.go:207] Starting reflector *v1.ConfigMap (30s) from k8s.io/client-go@v0.19.16/tools/cache/reflector.go:156
I0402 10:38:30.581893       1 main.go:189] cache of informer factories sync successfully
I0402 10:38:30.582011       1 backup_schedule_controller.go:70] Starting backup schedule controller
I0402 10:38:30.582013       1 restore_controller.go:71] Starting restore controller
I0402 10:38:30.582051       1 tidb_cluster_controller.go:98] Starting tidbcluster controller
I0402 10:38:30.582064       1 tidb_initializer_controller.go:69] Starting tidbinitializer controller
I0402 10:38:30.582051       1 tidb_ng_monitoring_controller.go:78] Starting tidbngmonitor controller
I0402 10:38:30.582065       1 backup_controller.go:78] Starting backup controller
I0402 10:38:30.582123       1 tidb_monitor_controller.go:65] Starting tidbmonitor controller
I0402 10:38:30.582181       1 pod_control.go:113] Starting tidbcluster pod controller
I0402 10:38:30.582016       1 dm_cluster_controller.go:91] Starting dmcluster controller
I0402 10:38:30.590591       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Role/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0402 10:38:30.594920       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ServiceAccount/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0402 10:38:30.606002       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create RoleBinding/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0402 10:38:30.615101       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Deployment/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0402 10:38:30.621112       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Service/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0402 10:38:30.969730       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd in TidbCluster test-cluster successful
I0402 10:38:31.366157       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd-peer in TidbCluster test-cluster successful
I0402 10:38:31.371682       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-pd-3731616 for controller TidbCluster/test-cluster successfully
I0402 10:38:31.399768       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"918", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-pd in TidbCluster test-cluster successful
I0402 10:38:31.425060       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:38:31.425090       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0402 10:38:31.495030       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: network is unreachable, service acto-namespace/test-cluster-pd has no endpoints
I0402 10:38:31.534551       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:38:31.534603       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:31.597703       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: network is unreachable, service acto-namespace/test-cluster-pd has no endpoints
I0402 10:38:31.640083       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:38:31.640128       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:31.696274       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: network is unreachable, service acto-namespace/test-cluster-pd has no endpoints
E0402 10:38:31.703611       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:33.495073       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: connection refused, service acto-namespace/test-cluster-pd has no endpoints
E0402 10:38:33.503673       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
I0402 10:38:37.372767       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1055", FieldPath:""}): type: 'Normal' reason: 'SuccessfulPatch' patch PV pvc-cd70be5b-3814-4ad8-a0ea-9faceb3e6269 in TidbCluster test-cluster successful
I0402 10:38:37.376882       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1055", FieldPath:""}): type: 'Normal' reason: 'SuccessfulPatch' patch PV pvc-5d0f233a-a62e-4852-a864-28ffecfefab2 in TidbCluster test-cluster successful
I0402 10:38:37.381686       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1055", FieldPath:""}): type: 'Normal' reason: 'SuccessfulPatch' patch PV pvc-ea672e41-cefb-4ba9-8e51-a363248bbabb in TidbCluster test-cluster successful
E0402 10:38:37.439644       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: connection refused
I0402 10:38:37.466508       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:38:37.466566       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:37.527924       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: connection refused
E0402 10:38:37.536731       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:37.609822       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: connection refused
I0402 10:38:37.631144       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:38:37.631188       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:37.831928       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: connection refused
I0402 10:38:37.877442       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:38:37.877494       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:38.433657       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.248.109:2379: connect: connection refused
E0402 10:38:38.441748       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:38:54.576747       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
E0402 10:38:54.590681       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
I0402 10:39:00.299401       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1180", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0402 10:39:00.315344       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:00.315420       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 0 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
I0402 10:39:00.402450       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1251", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0402 10:39:00.418165       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:00.418211       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 0 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
W0402 10:39:02.502846       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
I0402 10:39:02.511345       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1251", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0402 10:39:02.525512       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:02.525571       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 0 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
W0402 10:39:02.593714       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
I0402 10:39:02.601097       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1270", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0402 10:39:02.620733       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:02.620776       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 0 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
W0402 10:39:03.485057       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
E0402 10:39:03.492921       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0402 10:39:03.508221       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:03.508254       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
W0402 10:39:03.573850       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
I0402 10:39:03.581579       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1285", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0402 10:39:03.596054       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:03.596096       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 0 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
W0402 10:39:03.661408       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
E0402 10:39:03.668166       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0402 10:39:03.684825       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:03.684862       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
W0402 10:39:04.100208       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
E0402 10:39:04.109477       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0402 10:39:04.129701       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:04.129740       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
W0402 10:39:04.690102       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
I0402 10:39:04.697163       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1297", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0402 10:39:04.712477       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:04.712530       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 0 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
W0402 10:39:05.301277       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
E0402 10:39:05.311356       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0402 10:39:05.354485       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:05.354522       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
W0402 10:39:05.895889       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
E0402 10:39:05.902889       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0402 10:39:05.941307       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:05.941363       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
W0402 10:39:06.496985       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
I0402 10:39:06.503174       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1316", FieldPath:""}): type: 'Warning' reason: 'PDMemberUnhealthy' acto-namespace/test-cluster-pd-1(14783552960869691380) is unhealthy
I0402 10:39:06.542875       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:06.542920       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster's pd cluster is not healthy, healthy 0 / desired 3, replicas 3, failureCount 0, can't failover, requeuing
W0402 10:39:07.099905       1 pd_member_manager.go:364] PD member: [14256557501003737041] doesn't have a name, and can't get it from clientUrls: [[]], memberHealth Info: [{ 14256557501003737041 [] false}] in [acto-namespace/test-cluster]
E0402 10:39:07.108135       1 pd_failover.go:70] PD failover replicas (0) reaches the limit (0), skip failover
I0402 10:39:07.154582       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:07.154614       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0402 10:39:07.696081       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect to not leader
I0402 10:39:07.722616       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0402 10:39:07.722662       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
E0402 10:39:08.293387       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 500 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: redirect to not leader
E0402 10:39:08.300658       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed TidbCluster: acto-namespace/test-cluster .Status.PD.Synced = false, can't failover, requeuing
I0402 10:39:30.300653       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1339", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tikv-peer in  test-cluster successful
I0402 10:39:30.305322       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1339", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tikv-3831336 for controller TidbCluster/test-cluster successfully
I0402 10:39:30.312241       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1339", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tikv in  test-cluster successful
I0402 10:39:30.322950       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:30.322996       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:30.391149       1 tikv_member_manager.go:787] TiKV of Cluster acto-namespace/test-cluster not bootstrapped yet
I0402 10:39:30.397267       1 tikv_member_manager.go:885] TiKV of Cluster acto-namespace/test-cluster is not bootstrapped yet, no need to set store labels
I0402 10:39:30.441294       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:30.441333       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:30.525919       1 tikv_member_manager.go:787] TiKV of Cluster acto-namespace/test-cluster not bootstrapped yet
I0402 10:39:30.532012       1 tikv_member_manager.go:885] TiKV of Cluster acto-namespace/test-cluster is not bootstrapped yet, no need to set store labels
I0402 10:39:30.532854       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:35.503761       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1448", FieldPath:""}): type: 'Normal' reason: 'SuccessfulPatch' patch PV pvc-eff0a2e6-12c2-4f9e-9576-e3c22355a482 in TidbCluster test-cluster successful
I0402 10:39:35.507709       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1448", FieldPath:""}): type: 'Normal' reason: 'SuccessfulPatch' patch PV pvc-ede050c3-217e-4d7c-812b-46f5b4e33e47 in TidbCluster test-cluster successful
I0402 10:39:35.511200       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1448", FieldPath:""}): type: 'Normal' reason: 'SuccessfulPatch' patch PV pvc-839c079b-ad36-4590-aeb4-397800e74baa in TidbCluster test-cluster successful
I0402 10:39:35.592264       1 tikv_member_manager.go:787] TiKV of Cluster acto-namespace/test-cluster not bootstrapped yet
I0402 10:39:35.599110       1 tikv_member_manager.go:885] TiKV of Cluster acto-namespace/test-cluster is not bootstrapped yet, no need to set store labels
I0402 10:39:35.616882       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:35.616914       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:35.691807       1 tikv_member_manager.go:787] TiKV of Cluster acto-namespace/test-cluster not bootstrapped yet
I0402 10:39:35.697248       1 tikv_member_manager.go:885] TiKV of Cluster acto-namespace/test-cluster is not bootstrapped yet, no need to set store labels
I0402 10:39:35.698083       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:35.775871       1 tikv_member_manager.go:787] TiKV of Cluster acto-namespace/test-cluster not bootstrapped yet
I0402 10:39:35.989601       1 tikv_member_manager.go:885] TiKV of Cluster acto-namespace/test-cluster is not bootstrapped yet, no need to set store labels
I0402 10:39:36.011459       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:36.011502       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:36.391035       1 tikv_member_manager.go:787] TiKV of Cluster acto-namespace/test-cluster not bootstrapped yet
I0402 10:39:36.788510       1 tikv_member_manager.go:885] TiKV of Cluster acto-namespace/test-cluster is not bootstrapped yet, no need to set store labels
I0402 10:39:36.789873       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:37.611419       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:37.611456       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:38.412640       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:39:38.412682       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:39:39.194257       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
I0402 10:40:00.313069       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1567", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb-peer in  test-cluster successful
I0402 10:40:00.325209       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1567", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb in  test-cluster successful
I0402 10:40:00.328772       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1567", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tidb-6662316 for controller TidbCluster/test-cluster successfully
I0402 10:40:00.335074       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"be36129f-7c0a-4572-b0a6-eb9771e314f0", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"1567", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tidb in  test-cluster successful
I0402 10:40:00.368703       1 pv_control.go:192] PV: [pvc-cd70be5b-3814-4ad8-a0ea-9faceb3e6269] updated successfully, : acto-namespace/test-cluster
I0402 10:40:00.915876       1 pv_control.go:192] PV: [pvc-ea672e41-cefb-4ba9-8e51-a363248bbabb] updated successfully, : acto-namespace/test-cluster
I0402 10:40:01.513340       1 pv_control.go:192] PV: [pvc-ede050c3-217e-4d7c-812b-46f5b4e33e47] updated successfully, : acto-namespace/test-cluster
I0402 10:40:02.516237       1 pv_control.go:192] PV: [pvc-839c079b-ad36-4590-aeb4-397800e74baa] updated successfully, : acto-namespace/test-cluster
I0402 10:40:03.115870       1 pv_control.go:192] PV: [pvc-5d0f233a-a62e-4852-a864-28ffecfefab2] updated successfully, : acto-namespace/test-cluster
I0402 10:40:03.715055       1 pv_control.go:192] PV: [pvc-eff0a2e6-12c2-4f9e-9576-e3c22355a482] updated successfully, : acto-namespace/test-cluster
I0402 10:40:03.736264       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:40:03.835914       1 equality.go:87] Service spec diff for acto-namespace/test-cluster-tidb:   v1.ServiceSpec{
  	Ports: []v1.ServicePort{
  		{
  			... // 3 identical fields
  			Port:       4000,
  			TargetPort: {IntVal: 4000},
- 			NodePort:   0,
+ 			NodePort:   32670,
  		},
  		{
  			... // 3 identical fields
  			Port:       10080,
  			TargetPort: {IntVal: 10080},
- 			NodePort:   0,
+ 			NodePort:   31507,
  		},
  	},
  	Selector:  {"app.kubernetes.io/component": "tidb", "app.kubernetes.io/instance": "test-cluster", "app.kubernetes.io/managed-by": "tidb-operator", "app.kubernetes.io/name": "tidb-cluster"},
  	ClusterIP: "",
  	... // 12 identical fields
  }
I0402 10:40:03.835960       1 tidb_member_manager.go:442] Sync TiDB service acto-namespace/test-cluster-tidb, spec equal: false, annotations equal: true, label equal: true
I0402 10:40:03.913791       1 service_control.go:88] update Service: [acto-namespace/test-cluster-tidb] successfully, kind: , name: test-cluster
I0402 10:40:03.926117       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0402 10:40:04.963768       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:40:05.078056       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0402 10:40:05.124430       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:40:05.445397       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0402 10:40:05.491778       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:40:06.693191       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0402 10:40:20.522599       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully

